miden-tx/src/errors/tx_kernel_errors.rs

```diff
@@ -70,11 +70,11 @@ pub const ERR_ASSET_NOT_FUNGIBLE_ID: u32 = 131137;
 pub const ERR_ASSET_INVALID_AMOUNT: u32 = 131138;
 pub const ERR_ASSET_NOT_NON_FUNGIBLE_ID: u32 = 131139;
 pub const ERR_INVALID_NOTE_TYPE: u32 = 131140;
-pub const ERR_INVALID_NOTE_IDX: u32 = 131154;
+
+// TODO: fix the error values
 pub const ERR_NOTE_INVALID_TYPE_FOR_TAG: u32 = 131141;
-pub const ERR_NOTE_FUNGIBLE_MAX_AMOUNT_EXCEEDED: u32 = 131152;
-pub const ERR_NON_FUNGIBLE_ASSET_ALREADY_EXISTS: u32 = 131153;
 pub const ERR_NOTE_INVALID_TAG_PREFIX_FOR_TYPE: u32 = 131141;
```

Comment:
I'm assuming fixing it here would require too much change, right?

Reply:
I was thinking to fix this in the #712 PR, but for now I'm just not sure whether it is a typo in error code (so I can just change it to the not used value) or this error was added twice by mistake (so I can just remove one error): I wanted to look through the changes history to make sure.